### ISSUE TITLE
Return errors instead of outputing them to stderr

### ISSUE THREAD

**Issue by 1999** [torifat/better-avj-errors#1]
My proposal is to return a structure like this:

```json
{
  "error": "You're using invalid field FOO",
  "line": 14,
  "column": 75,
  "suggestion": "Maybe you meant BAR?"
}
```

**Comment by 1999** [comment]
https://github.com/torifat/better-avj-errors/pull/3/files

**Comment by torifat** [comment]
Closed via #3
